### PR TITLE
tweak: slightly improve `help lib.install`

### PR DESCRIPTION
**Pull request by mitchellwrosen**
## Overview

This PR improves the `help lib.install` output a bit.

2024-05-20-085134_1284x276_scrot

### CODE REVIEW

File: unison-cli/src/Unison/CommandLine/InputPatterns.hs [hunk 1]
```diff
@@ -1238,10 +1238,23 @@ libInstallInputPattern =
...
+              "The"
+                <> makeExample' libInstallInputPattern
+                <> "command installs a dependency into the `lib` namespace.",
+            "",
+            P.wrapColumn2
+              [ ( makeExample libInstallInputPattern ["@unison/base/releases/latest"],
+                  "installs the latest release of `@unison/base`"
+                ),
+                ( makeExample libInstallInputPattern ["@unison/base/releases/3.0.0"],
+                  "installs version 3.0.0 `@unison/base`"
```
Comment (aryairani) [inline unison-cli/src/Unison/CommandLine/InputPatterns.hs]:
```suggestion
                  "installs version 3.0.0 of `@unison/base`"
```

**Comment by aryairani** [comment]
Looks good, thanks. I suggested a fix for one typo, but we need to re-run transcripts for it.

**Comment by aryairani** [comment]
Sorry, I was wrong about the transcripts.

**Review by aryairani — APPROVED** [review]

**Comment by hojberg** [comment]
@mitchellwrosen does the new `lib.install` support just giving it a project reference: `lib.install @unison/base` or is the `/releases/latest` bit required? (I need to make sure the modals in Share reflect this correctly)

**Comment by mitchellwrosen** [comment]
@hojberg The branch is optional, and if omitted, is equivalent to `/releases/latest`

**Comment by mitchellwrosen** [comment]
(heh that's missing from the help output, guess I have to do this PR again but better)

**Pull request merged**